spec/controllers/reports/regions_controller_spec.rb

```diff
@@ -600,4 +633,40 @@
         params: {id: region.slug, report_scope: "state", format: "csv", period: period.value}
     end
   end
+
+  describe "#diabetes" do
+    let(:facility) { create(:facility, facility_group: facility_group_1, enable_diabetes_management: true) }
+    let(:facility_region) { create(:region, region_type: :facility, source: facility, reparent_to: Region.root) }
+
+    before do
+      sign_in(cvho.email_authentication)
+      Flipper.enable(:diabetes_management_reports)
```

Comment:
Let's move this flipper enable to inside the `it`s, that way we don't have to disable it to override on line 664.